### ISSUE TITLE
Improving `.travis.yml`

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#74]
Running PUBLISHING to PyPI stage only after merge. When PR - only run unit-tests and build steps.

**Comment by Tim55667757** [comment]
Separate PR and normal build was implemented.

Debug build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/257979587
Debug PR build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/257979802
